Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -526,22 +525,19 @@ public class BrowserViewController: UIViewController, BrowserViewControllerDeleg
     updateWidgetFavoritesData()
     
     Task { @MainActor in
+      self.setupTabs()
+      
       await ContentBlockerManager.shared.loadBundledResources()
       await ContentBlockerManager.shared.loadCachedCompileResults()
       
       // Load cached data
       // This is done first because compileResources and compilePendingResource need their results
-      async let filterListCache: Void = await FilterListResourceDownloader.shared.loadCachedData()
-      async let adblockResourceCache: Void = await AdblockResourceDownloader.shared.loadCachedData()
-      _ = await (filterListCache, adblockResourceCache)
+      await FilterListResourceDownloader.shared.loadCachedData()
+      await AdblockResourceDownloader.shared.loadCachedData()
```

Comment:
There may still be some benefit in doing this in parallel as we still want this all to happen as fast as possible so we can be faster than the user